pkg/cmd/pulumi/operations/up.go

```diff
@@ -599,6 +602,22 @@ func NewUpCmd() *cobra.Command {
 			logging.V(7).Infof("PULUMI_SUPPRESS_COPILOT_LINK=%v", env.SuppressCopilotLink.Value())
 			opts.Display.ShowLinkToCopilot = !env.SuppressCopilotLink.Value()
 
+			// Handle copilot-summary flag and environment variable
+			// If flag is explicitly set (via command line), use that value
+			// Otherwise fall back to environment variable, then default to false
```

Comment:
Nit: we wrap at 120 in the codebase; not sure if it's possible to apply that consistently in this PR

Reply:
`golines --shorten-comments` !

Reply:
Too hard. vscode > Rewrap